projects/cyrusimap.org/cyrus-sasl/package.yml

```diff
@@ -0,0 +1,35 @@
+distributable:
+  url: https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-{{version}}/cyrus-sasl-{{version}}.tar.gz
+  strip-components: 1
+
+display-name: cyrus-sasl
+
+versions:
+  github: cyrusimap/cyrus-sasl
+  strip: /^cyrus-sasl-/
+
+platforms:
+  - darwin
+  - linux/x86-64
```

Comment:
is this correct? please provide a `FIXME` comment so we know what the blocker is.

Reply:
It's not correct. I can't remember why I put that there. Or maybe I copied it from some other project and didn't remove it. I can't remember. Anyway, I've committed a fix.